CHANGELOG.md

```diff
@@ -30,6 +30,10 @@ update the Unreleased link so that it compares against the latest release tag.
 
 ## [Unreleased]
 
+### Fixed
+
+ - Removed the library file names from the full library path and file name before it was added to ALLLIBDIRS for the loader -L flag. [#3886](https://github.com/USGS-Astrogeology/ISIS3/issues/3986)
+
```

Comment:
This isn't a change that will impact users, I don't think it should be in the changelog.

Reply:
Sounds good, I'll remove it.